pkg/reconciler/kubernetes/progress/progress.go

```diff
@@ -202,12 +202,7 @@ func (pt *Tracker) statefulsetInState(inState State, object *resource) (bool, er
 		if err != nil {
 			return false, err
 		}
-		for _, condition := range statefulSet.Status.Conditions {
-			if condition.Status != v1.ConditionTrue {
-				return false, nil
-			}
-		}
-		return true, err
+		return statefulSet.Status.ReadyReplicas == statefulSet.Status.Replicas, nil
```

Comment:
But isn't a stateful set already treated as ready when 1 of the is ready (like `statefulSet.Status.ReadyReplicas > 0`) ?

Reply:
I would say its kind of misleading from the cli point of view as it would say `kyma is installed` although some pods are still starting. up